03week/towersOfHanoi.js

```diff
@@ -13,29 +13,68 @@ let stacks = {
   c: []
 };
 
+
+
 function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
-
-function movePiece() {
+/* this function selects the last object in the array you have selected and the last object in the array
+you want to put it in and sets them to variables. it then
+*/
+function movePiece(startStack,endStack) {
   // Your code here
 
-}
+  let o = stacks[startStack]
```

Comment:
short variable names like this will make it harder when your programs get more involved. You might want to consider picking variable names that are a little more descriptive.